api/proxy/servers/arbitrum_altda/cli.go

```diff
@@ -0,0 +1,51 @@
+package arbitrum_altda
+
+import (
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	ListenAddrFlagName = "arbitrum-da.addr"
```

Comment:
nit: might be better to use `host` instead of `addr`? Since `localhost` is a valid addr for example.